library/types/src/modules/URLRecode.rb

```diff
@@ -47,7 +47,7 @@ def escape(input, safe_chars)
       return nil if input.nil?
       input.gsub(/[^#{safe_chars}]/) do |unicode_char|
         escaped = ""
-        unicode_char.each_byte { |b| escaped << sprintf("%%%02x", b) }
+        unicode_char.each_byte { |b| escaped << format("%%%02x", b) }
```

Comment:
yep, new version of rubocop prefer format over sprintf. I think it can be switched to prefer sprintf over format. I think goal is to use just one alias to avoid confusion.